tests/conftest.py

```diff
@@ -26,9 +26,7 @@ async def set(self, id, data) -> None:
 
     async def get(self, id) -> dict:
         """Mock `get()` method."""
-        import json
-
-        return json.dumps(self.obj[id])
+        return self.obj[id]
```

Comment:
Maybe it should be `json.loads(json.dumps(self.obj[id]))`? Just to ensure it can be JSON encoded as it would be in the actual endpoint?